index.html

```diff
@@ -6,9 +6,13 @@
       <link href="index.css" media="screen" rel="stylesheet" type="text/css"/>
   </head>
 
-  <body>
+  <body onload='changeTime()'>
```

Comment:
This should be in a [`$( document ).ready()`](https://learn.jquery.com/using-jquery-core/document-ready/) function instead of an `onload` attribute. 

This is so the coding elements can be all in all in one place rather than jumping between html and JS files.